api/chaos/experimentation/v1/experiment.proto

```diff
@@ -10,17 +10,17 @@ option go_package = "experimentationv1";
 message Experiment {
   enum Status {
     // Unspecifiedf Status.
-    UNSPECIFIED = 0;
+    STATUS_UNSPECIFIED = 0;
```

Comment:
The syntax is a bit redundant here since it would look like `Status.Status_Unspecified`

Reply:
yes but it makes more sense in Go/TS layer (i.e., `Experiment_STATUS_UNSPECIFIED`)